src/js/components/dashboard/DashboardTable.jsx

```diff
@@ -389,7 +394,7 @@ export default class DashboardTable extends React.Component {
 
         return (
             <div className="usa-da-submission-list">
-                <div>
+                <div className={this.tableClasses()}>
                     <FormattedTable
```

Comment:
So you can do this:

```javascript

classNames({
'name-of-class':boolean-producing-javascript-expression
})
```

Reply:
Also, I don't think you need to put this logic in a class method, you can just do

`<div className={`some-class-thats-always-here ${classNames({ not-always-here: props.condition })}`}>`

Reply:
Yeah, [classnames](https://www.npmjs.com/package/classnames) provides a clean way to handle multiple classnames with logical operations

`var classNames = require('classnames');
 
var Button = React.createClass({
  // ...
  render () {
    var btnClass = classNames({
      btn: true,
      'btn-pressed': this.state.isPressed,
      'btn-over': !this.state.isPressed && this.state.isHovered
    });
    return <button className={btnClass}>{this.props.label}</button>;
  }
});`

Reply:
Perfect, lets just double check these new modules are good to go, then I'll approve.